src/bundle/Controller/ContentTypeGroupController.php

```diff
@@ -128,6 +131,14 @@ public function createAction(Request $request): Response
                     'ibexa_content_type'
                 );
 
+                if ($form->getClickedButton() instanceof Button
```

Comment:
Is this check only to avoid phpstan error? if yes, then maybe it should be checked against `\Symfony\Component\Form\FormInterface` as this is what provides getName() function.

Reply:
This is how it was already done in the other place and I think it is fine. In some places, there are just exceptions added to phpstan baseline.